vector/src/main/java/im/vector/app/features/lifecycle/VectorActivityLifecycleCallbacks.kt

```diff
@@ -68,11 +68,7 @@ class VectorActivityLifecycleCallbacks constructor(private val popupAlertManager
 
             // Get all activities from PermissionController module
             // See https://source.android.com/docs/core/architecture/modular-system/permissioncontroller#package-format
-<<<<<<< HEAD
             if (Build.VERSION.SDK_INT > Build.VERSION_CODES.S) {
-=======
-            if (Build.VERSION.SDK_INT > Build.VERSION_CODES.S_V2) {
```

Comment:
I think we can take this one, I don't see a reason to have a difference here

Reply:
Build.VERSION_CODES.S_V2 doesn't exist in this project